VirtoCommerce.Storefront/Controllers/AccountController.cs

```diff
@@ -309,7 +311,23 @@ public async Task<ActionResult> Login([FromForm] Login login, [FromQuery]string
                 return View("customers/login", WorkContext);
             }
 
-            if (!new CanUserLoginToStoreSpecification(user).IsSatisfiedBy(WorkContext.CurrentStore) || new IsUserSuspendedSpecification().IsSatisfiedBy(user))
+            if (!new CanUserLoginToStoreSpecification(user).IsSatisfiedBy(WorkContext.CurrentStore))
```

Comment:
Why we can't do this by passing a returnUrl in the query string in form post action?
e.g `{{ '/' | store_absolute_url }}`

Reply:
@tatarincev Because code will not pass to this action, it will not allow you to login to this store